### DISCUSSION TITLE
Ping Goes Poof

### DISCUSSION THREAD

**Discussion by giscus** [davidsneighbour/kollitsch.dev#335] [Comments]
# Ping Goes Poof

This is the online playground of Patrick Kollitsch, David's Neighbour, in Germany produced, digital native living & working on Koh Samui in Thailand.

https://kollitsch.dev/blog/2023/ping-goes-poof/

**Comment by brycewray** [comment]
Interesting stuff. I’d love to see the code behind that module. I’m not much of a Hugo Modules guy, so I’d rather just see how you do it. :D Is there a direct link to it?

...other than https://github.com/davidsneighbour/hugo-blockify/blob/main/layouts/sitemap.xml, of course?

**Reply by davidsneighbour** [reply]
I am updating those modules (head, sitemap, robots, and maybe social) today, then you can see it. Probably only that file and then some config options in `config/_default/params.toml` that start with `[dnb.sitemap]`.

**Reply by davidsneighbour** [reply]
[Here](https://github.com/davidsneighbour/hugo-blockify/blob/25e1c32cc221f6efb43ed76d3ab98a87ecbe95c2/config/_default/params.toml#L379) and [here](https://github.com/davidsneighbour/hugo-blockify/blob/main/layouts/partials/func/isSitemapped.html).
